app/src/main/java/org/fossasia/susi/ai/skills/help/HelpFragment.kt

```diff
@@ -0,0 +1,28 @@
+package org.fossasia.susi.ai.skills.help
+
+import android.os.Bundle
+import android.support.v4.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import org.fossasia.susi.ai.R
+import org.fossasia.susi.ai.skills.SkillsActivity
+
+class HelpFragment : Fragment() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        // Inflate the layout for this fragment
+        val thisActivity = activity
+        if (thisActivity is SkillsActivity) thisActivity.title = getString(R.string.action_help)
```

Comment:
Why is this needed?

Reply:
Sir, this is required to change the title of the page from "SUSI.AI Skills" to Help

Reply:
Why is the cast to Skills Activity needed?